jschon/vocabulary/__init__.py

```diff
@@ -105,6 +121,17 @@ def __str__(self) -> str:
 KeywordClass = Type[Keyword]
 
 
+class AnnotationMixin(Keyword):
+    def evaluate(self, instance: JSON, result: Result) -> None:
+        result.annotate(self.json.data)
+
+
+class AnnotationKeyword(AnnotationMixin):
```

Comment:
Please add `'AnnotationKeyword'` to the list of exported module members in `__all__`.

Reply:
Oop!  I forget about `__all__` as I've never used it much (in the last really big Python project I worked on we banned using `import *` instead).